Makefile

```
@@ -18,9 +18,8 @@ test:
 test-coverage:
 	@mkdir -p .coverage
 	@go test `go list ./... | grep -Ev "diodepb|examples|internal"` -race -cover -json -coverprofile=.coverage/cover.out.tmp ./... | tparse -format=markdown > .coverage/test-report.md
```

Have we tried excluding `ingester.go` from the go list here before it gets into cover?
```suggestion
	@go test `go list ./... | grep -Ev "diodepb|examples|internal|ingester.go"` -race -cover -json -coverprofile=.coverage/cover.out.tmp ./... | tparse -format=markdown > .coverage/test-report.md
```

---

it didn't work =\ same result in md file generated